config/locales/en.yml

```diff
@@ -16,6 +16,16 @@ en:
         resource_type: "Resource type"
         updated_at: "Updated"
   active_admin:
+    toggle_section:
+      show: "Show"
+      hide: "Hide"
+    toggle_user_menu:
+      sign_out: "Sign Out"
+    toggle_dark_mode:
+      dark_mode: "Dark Mode"
+      light_mode: "Light Mode"
+    toggle_main_navigation_menu:
+      main_navigation: "Main Navigation"
```

Comment:
Hi, these are unwanted changes?

I do not expect changes in English locale file. If we need a change here, then we have a different bug.

Please find the following below:

```
    toggle_dark_mode: Toggle dark mode
    toggle_main_navigation_menu: Toggle main navigation menu
    toggle_section: Toggle section
    toggle_user_menu: Toggle user menu
```

You may want to run `i18n-tasks add-missing` to populate your locale of interest with missing locales, and then start from there